galasa-parent/dev.galasa.framework.api.users/src/main/java/dev/galasa/framework/api/users/internal/routes/UsersRoute.java

```diff
@@ -52,44 +57,97 @@ public HttpServletResponse handleGetRequest(String pathInfo, QueryParameters que
 
         logger.info("UserRoute: handleGetRequest() entered.");
 
-        validateQueryParam(queryParams, request.getServletPath());
+        List<UserData> usersList = new ArrayList<>();
+        String payloadContent = "{}";
```

Comment:
The OpenAPI spec says that `GET /users` returns an array, so instead of defaulting to an empty JSON object, the default should be an empty array (e.g. `new JsonArray()`)